src/common.ts

```diff
@@ -8,11 +10,18 @@ export interface TransactionOptions {
 }
 export interface CreateSessionOptions {
   name: string;
-  labels: {[key: string]: string};
+  labels: { [key: string]: string; };
   createTime: GetTimestamp;
   approximateLastUseTime: GetTimestamp;
 }
 export interface GetTimestamp {
-  nanos: number;
-  seconds: number;
+  nanos: number; seconds: number;
+};
+
+export interface RowCountsServiceError {
+  //tslint:disable-next-line no-any
+  rowCounts?: number[] | ((rowCounts: any, arg1: never[]) => any);
```

Comment:
Where is `rowCounts` used?

Reply:
Removed `RowCountsServiceError` as it's not needed any more